Autopilot/AttitudeManager/timer.c

```diff
@@ -36,10 +36,10 @@ void initTimer4(){
     T4CONbits.TGATE = 0; // Disable Gated Timer mode
     T4CONbits.TCKPS = 0b10; // Select 1:64 Prescaler
     TMR4 = 0x00; // Clear timer register
-    PR4 = 642; // Load the period value
-    IPC6bits.T4IP = 0x01; // Set Timer1 Interrupt Priority Level
-    IFS1bits.T4IF = 0; // Clear Timer1 Interrupt Flag
-    IEC1bits.T4IE = 1; // Enable Timer1 interrupt
+    PR4 = 625; // Load the period value
```

Comment:
Theres a flag for this value. I beleive i named it T2_TIMER_TICKS_TO_MS however it applies to both timers. rename and adjust that define and set it to PR4, since what you're doing only updates Timer 4, but not Timer 2